### PR TITLE
TDD project

### PR DESCRIPTION
**Pull request by ogagaoghene**
# Test Driven Development Project

## Description 
Write tests for Solver class and its methods. Each test case addresses an example or a unique test case which makes the failing tests to pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,105 @@
...
+
+### Clone this repository
+
+```bash
+git clone https://github.com/ogagaoghene/TDD_Project.git
+```
+### Move into the directory with
+
+```bash
+cd oop_school_library_ruby
```
Comment (Deepakdanger) [inline README.md]: - [ ] [OPTIONAL] Here kindly check the readme file & correct the file name.

**Review by Deepakdanger — APPROVED** [review]
Hi @TEAM, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-  [ ] The tests folder name should be `spec` instead of `specs`.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ogagaoghene) [reply README.md]: I will do as requested. Thanks alot

Comment (ogagaoghene) [reply README.md]: @Deepakdanger I will do as requested. Thanks for your professional code review

**Pull request merged**